src/components/characters/Character.jsx

```diff
@@ -1,19 +1,22 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
 import PropTypes from 'prop-types';
 
-const Character = ({ name, affiliation, photoUrl }) => {
+const Character = ({ _id, name, affiliation, photoUrl }) => {
```

Comment:
yep. that'll do it. Make sure to check the API to see what's being returned. You can also munge the data but definitely check the shape and properties that are returned.